unified-runtime/scripts/core/exp-bindless-images.yml

```diff
@@ -212,6 +212,10 @@ etors:
     desc: "Win32 NT handle"
   - name: WIN32_NT_DX12_FENCE
     desc: "Win32 NT DirectX 12 fence handle"
+  - name: TIMELINE_FD
+    desc: "Timeline file descriptor"
```

Comment:
```suggestion
    desc: "Timeline semaphore file descriptor"
```

Reply:
I have updated all the enum descs to make sure they all include semaphore, the type of semaphore and whether they are used for vulkan or DX12.